src/modules/salesforce/salesforce.service.test.ts

```diff
@@ -105,6 +109,10 @@ describe('SalesforceService', () => {
         Party_URN__c: null,
         D_B_Number__c: null,
         Company_Registration_Number__c: '12341234',
+        CCM_Credit_Risk_Rating_Date__c: '2007-03-27',
+        CCM_Credit_Risk_Rating__c: 'B+',
+        CCM_Loss_Given_Default__c: 50,
+        CCM_Probability_of_Default__c: null,
```

Comment:
Can we add some assertions with edge values.
PoD: 100
Risk Rating: AAA+
LGD: 100